scripts/PBAB+Collabs/flex-engine-demo/1_flex_engine_demo_contract_suite_deployer_staging_goerli.ts

```diff
@@ -0,0 +1,257 @@
+  // );
+  // // DA Lin V1
+  // const MinterDALin__factory = new MinterDALinV1__factory(deployer);
+  // const minterDALin = await MinterDALin__factory.deploy(
+  //   genArt721CoreFlex.address,
+  //   minterFilter.address
+  // );
+  // await minterDALin.deployed();
+  // console.log(`Minter DA Lin V1 for FLEX deployed at ${minterDALin.address}`);
+  // // DA Exp V1
+  // const MinterDAExp__factory = new MinterDAExpV1__factory(deployer);
+  // const minterDAExp = await MinterDAExp__factory.deploy(
+  //   genArt721CoreFlex.address,
+  //   minterFilter.address
+  // );
+  // await minterDAExp.deployed();
+  // console.log(`Minter DA Exp V1 for FLEX deployed at ${minterDAExp.address}`);
+
+  // //////////////////////////////////////////////////////////////////////////////
+  // // DEPLOYMENT ENDS HERE
+  // //////////////////////////////////////////////////////////////////////////////
+
+  // //////////////////////////////////////////////////////////////////////////////
+  // // SETUP BEGINS HERE
+  // //////////////////////////////////////////////////////////////////////////////
+  // let tx = null;
+  // // Allowlist the Minter on the Core contract.
+  // tx = await genArt721CoreFlex
+  //   .connect(deployer)
+  //   .addMintWhitelisted(minterFilter.address);
+  // await tx.wait();
+  // console.log(`Allowlisted the Minter Filter on the Core contract.`);
+  // delay(EXTRA_DELAY_BETWEEN_TX);
+
+  // // Update the Art Blocks Address.
+  // tx = await genArt721CoreFlex
+  //   .connect(deployer)
+  //   .updateRenderProviderAddress(artblocksAddress);
+  // await tx.wait();
+  // console.log(`Updated the artblocks address to: ${artblocksAddress}.`);
+  // delay(EXTRA_DELAY_BETWEEN_TX);
+
+  // // Allowlist AB staff (testnet only)
+  // if (
+  //   network.name == "ropsten" ||
+  //   network.name == "rinkeby" ||
+  //   network.name == "goerli"
+  // ) {
+  //   console.log(`Detected testnet - Adding AB staff to the whitelist.`);
+  //   const devAddresses = [
+  //     "0xB8559AF91377e5BaB052A4E9a5088cB65a9a4d63", // purplehat
+  //     "0x3c3cAb03C83E48e2E773ef5FC86F52aD2B15a5b0", // dogbot
+  //     "0x0B7917b62BC98967e06e80EFBa9aBcAcCF3d4928", // ben_thank_you
+  //   ];
+  //   for (let i = 0; i < devAddresses.length; i++) {
+  //     tx = await genArt721CoreFlex
+  //       .connect(deployer)
+  //       .addWhitelisted(devAddresses[i]);
+  //     await tx.wait();
+
+  //     console.log(`Allowlisted ${devAddresses[i]} on the Core contract.`);
+  //     delay(EXTRA_DELAY_BETWEEN_TX);
+  //   }
+  // }
+
+  // // Allowlist new Minters on MinterFilter.
+  // tx = await minterFilter
+  //   .connect(deployer)
+  //   .addApprovedMinter(minterSetPrice.address);
+  // await tx.wait();
+  // console.log(`Allowlisted minter ${minterSetPrice.address} on minter filter.`);
+  // delay(EXTRA_DELAY_BETWEEN_TX);
+  // tx = await minterFilter
+  //   .connect(deployer)
+  //   .addApprovedMinter(minterSetPriceERC20.address);
+  // await tx.wait();
```

Comment:
I have a strong preference that we don't include commented out code in PRs, can we just delete all of this commented out code?

Reply:
I think this is not meant to be commented out, and was just for some deployment debugging re: needed to generate typechain

Reply:
I think this is not meant to be commented out, and was just for some deployment debugging re: needed to generate typechain

Reply:
uncommented 👍